queenbee/schema/workflow.py

```diff
@@ -97,6 +127,50 @@ def nodes_links(self):
         return {'nodes': nodes, 'links': links}
 
 
+def hydrate_templates(workflow, wf_value=None):
```

Comment:
This is nice! So you are basically making a static version of the workflow with values provided by user. I think it will be helpful to add more description on why or when people should use this.

Reply:
Will do. My thinking was that it would mostly be used by the plugins when writing/running workflows.